### DISCUSSION TITLE
Unable to upload ID, as it stuck on Processing

### DISCUSSION THREAD

**Discussion by dipanadhi** [community/community#102863] [GitHub Education] [answered]
### Hi everyone,

I am trying to apply for student developer pack, however, when I try to upload my ID, it stuck on processing and when I open the console, it is showing payload is too large. However, my ID image size is 3024*4032 and only 1.3MB, which is with in the limit of 10MB. 

![image](https://github.com/community/community/assets/158131283/fcd389ca-d0c2-4b25-bd0c-c79a4f460032)

**Comment by mw-xyz** [comment, marked as answer]
Try resizing to a smaller resolution e.g. this is 1.33:1 so perhaps go for a smaller one. Let me know if this helps!

**Reply by mishmanners** [reply]
Make sure the file is less than 150kB 😉
